projects/media-viewer/src/lib/viewers/image-viewer/image-viewer.component.scss

```diff
@@ -60,3 +60,7 @@ img {
   -o-transform-origin: left top;
   transform-origin: left top;
 }
+
+.commentSummaryHeader {
+  top: 7px;
```

Comment:
is there a reason this is done here and not in the header component?

Reply:
yes because the comment layout is a bit messed up... i hope that will get fixed in the next PR you and veli working on...